opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectBookmark.jsx

```diff
@@ -162,7 +165,7 @@ class StixDomainObjectBookmarkComponent extends Component {
                 />
               </Avatar>
             )}
-            title={node.name}
+            title={renderThreatActorIndividual(node)}
```

Comment:
Same question here as for GenericAttackCard : is this method generic or specific for Threat Actors Individuals ? StixDomainObjectBookmark is a shared component.